.github/workflows/ci.yml

```diff
@@ -240,13 +242,13 @@ jobs:
         run: ${{ format('./test_bitcoin{0}', matrix.file_ext) }} -p
 
   test-gtest:
-    name: GoogleTest tier ${{ matrix.tier }} platform ${{ matrix.platform }}
+    name: GoogleTest tier ${{ matrix.tier }} platform ${{ matrix.platform }} - sharding ${{ matrix.gtest_sharding }} of 2
     needs: [matrices, build]
     runs-on: ${{ matrix.test_os }}
     continue-on-error: ${{ matrix.tier != 1 }}
     strategy:
       matrix:
-        include: ${{ fromJson(needs.matrices.outputs.test_matrix) }}
+        include: ${{ fromJson(needs.matrices.outputs.gtest_matrix) }}
```

Comment:
Instead of defining a separate `gtest_matrix`, we shoulkd be able to add a sharding axis to the matrix. This should be combined with the specific matrix entries added via `include`: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations.

Reply:
good idea, let me try it with that